crates/cxx-qt-lib/src/gui/qimage.rs

```diff
@@ -30,9 +132,26 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qimage_init_from_data"]
         fn qimageInitFromData(data: &[u8], format: &str) -> QImage;
+
+        #[doc(hidden)]
+        #[rust_name = "qimage_cache_key"]
+        fn qimageCacheKey(image: &QImage) -> i64;
     }
 }
+/*
+#[cfg(qt_version_major = "6")]
+#[cxx::bridge]
+mod ffi_qt_6 {
```

Comment:
either need to figure out what is happening or remove the commented code :-)

Reply:
Yep for the moment it failed to compile :( No idea yet.

Reply:
So perhaps I can remove commented code and qimage support is improved :)

Reply:
Can you split this into a separate issue and link how the `cfg` stuff should work

Reply:
https://github.com/KDAB/cxx-qt/issues/765